jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridView.java

```diff
@@ -249,9 +248,23 @@ public String getValidKeyForAdditionalListGrid( String baseName ) {
         return newListGrid;
     }
 
-    protected ExtendedPagedTable<T> createExtendedPagedTable(final GridGlobalPreferences preferences,
+    protected ExtendedPagedTable<T> createExtendedPagedTable(final GridGlobalPreferences defaultPreferences,
                                                              final String key) {
-        return TAB_SEARCH.equals(key) ? createAdvancedSearchTable(preferences) : new ExtendedPagedTable<T>(preferences);
+        GridGlobalPreferences pref;
+        ExtendedPagedTable<T> table ;
+        if(TAB_SEARCH.equals(key)){
+            pref = new GridGlobalPreferences(defaultPreferences.getKey() + key,
```

Comment:
you are absolutely right, it' s only necessary  add the defaultPreferences.getKey() to the grid preference key at  the TAB_SEARCH, to avoid collision with other view  Tab search stored preferences.
@jhrcek  changed. Can you take a look? thanks